qmxgraph/api.py

```diff
@@ -870,31 +869,40 @@ def is_cells_connectable(self):
     def run_layout(self, layout_name):
         return self.call_api('runLayout', layout_name)
 
-    def call_api(self, fn, *args):
+    def call_api(self, fn: str, *args, sync=True, check_api=True):
```

Comment:
Document new params (sync and check_api)

Reply:
Added TODO (I will remove the parameter)